client/src/components/Searchbar.js

```diff
@@ -15,14 +18,38 @@ const Input = styled.input`
   min-width: 284px;
   background-image: url(${SearchIcon});
   background-repeat: no-repeat;
-  background-position: 8px 7px;
+  background-position: 12px 10px;
   font: var(--font-robotolight);
 `;
 
-export default function Searchbar() {
+const ClearButton = styled.div`
+  position: absolute;
+  top: 57px;
+  right: 57px;
+  background: transparent;
+  border: none;
+  &:focus {
+    outline: none;
+  }
+`;
+
+export default function Searchbar({ onSubmit, onClear, ...props }) {
+  const handleClear = (e) => {
+    e.preventDefault();
+    console.log("onClear", onClear);
+    onClear();
+  };
   return (
-    <InputContainer>
-      <Input placeholder="Search for any tasting" />
-    </InputContainer>
+    <InputForm onSubmit={onSubmit}>
+      <Input placeholder="Search for any tasting" {...props} />
+      <ClearButton onClick={handleClear}>
+        <img src={CancelIcon} alt="Cancel" />
+      </ClearButton>
+    </InputForm>
   );
 }
+
+Searchbar.propTypes = {
+  onSubmit: PropTypes.string.isRequired,
+  onClear: PropTypes.string.isRequired,
```

Comment:
Be careful here.  Neither of these are a string. 
```suggestion
  onSubmit: PropTypes.func.isRequired,
  onClear: PropTypes.func.isRequired,
```
In general, onXXXX always suggest an event. A string could be named eg.: query, dataInput, value or something else